### PR TITLE
[CDS-54856]: Adding OCI helm go files

### PR DESCRIPTION
**Pull request by namantalaycha10**
## Comment Triggers

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`

### CODE REVIEW

File: harness/cd/graphql/types.go [hunk 1]
```diff
@@ -604,6 +604,10 @@ type HTTPHelmRepoConnector struct {
 	Connector
 }
 
+type OCIHelmRepoConnector struct {
```
Comment (rathodmeetsatish) [inline harness/cd/graphql/types.go:607]: cd folder is for cg we dont need this change

File: harness/nextgen/enum_oci_helm_auth_types.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+type OciHelmAuthType string
+
+var OciHelmAuthTypes = struct {
+	Anonymous        OciHelmAuthType
+	UsernamePassword OciHelmAuthType
+}{
+	Anonymous:        "Anonymous",
+	UsernamePassword: "UsernamePassword",
+}
+
```
Comment (rathodmeetsatish) [inline harness/nextgen/enum_oci_helm_auth_types.go:12]: please refer harness/nextgen/enum_service_now_auth_types.go we need to add 1 block here

**Review by rathodmeetsatish — CHANGES_REQUESTED** [review]

File: harness/cd/graphql/types.go (hunk 1 above)
Comment (namantalaycha10) [reply harness/cd/graphql/types.go]: Done

File: harness/nextgen/enum_oci_helm_auth_types.go (hunk 2 above)
Comment (namantalaycha10) [reply harness/nextgen/enum_oci_helm_auth_types.go:12]: Done

File: harness/nextgen/model_oci_helm_username_password.go [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+ *
+ * API version: 3.0
+ * Contact: contact@harness.io
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+package nextgen
+
+// This contains oci helm username password details
+type OciHelmUsernamePassword struct {
+	Username string `json:"username,omitempty"`
```
Comment (rathodmeetsatish) [inline harness/nextgen/model_oci_helm_username_password.go:14]: please do cmd+s on this file to fix indendation

File: harness/nextgen/model_oci_helm_connector.go [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+ *
+ * API version: 3.0
+ * Contact: contact@harness.io
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+package nextgen
+
+// This contains Oci helm connector details
+type OciHelmConnector struct {
+	ConnectorType string `json:"connectorType"`
```
Comment (rathodmeetsatish) [inline harness/nextgen/model_oci_helm_connector.go:14]: please do cmd+s on this file to fix indendation

**Review by rathodmeetsatish — CHANGES_REQUESTED** [review]

**Review by rathodmeetsatish — APPROVED** [review]

**Review by abosii-harness — APPROVED** [review]

**Pull request merged**